File: primerTrimestre/introduccion_programacion_python/3repetitivas_py/ej9rep.py
# Programa: ej9rep.py
# Propósito: Mostrar en pantalla los N primero número primos. Se pide por
# teclado la cantidad de números primos que queremos mostrar.
# Autor: Antonio Quesada
# Fecha: 30/10/2019.
#
# Variables a usar:
#   * cantidad
#    
#
# Petición de datos.
cantidad = int(input("Introduce la cantidad de números primos que quieres mostrar: "))

#Ciclo for y salida.
#Debe mostrar con un bucle for una x cantidad de numeros a partir de 2 que es el primer número primo.
for primo in range(2,cantidad+cantidad):
    
    #Si el numero primo es divisible por el mismo o por 1, se cumple que es primo y lo imprime.
    if (primo%2!=0 or primo==2) and (primo%3!=0 or primo==3) and (primo%5!=0 or primo==5) and (primo%7!=0 or primo==7):
        print(primo)  

File: primerTrimestre/java basico/3repetitivas/Ej9rep.java
/**Mostrar en pantalla los N primero número primos. Se pide por teclado la cantidad de números
 * primos que queremos mostrar.
 *
 * @author Antonio Quesada
 */

import java.util.Scanner;

public class Ej9rep {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    int cantidad;
    
    //Petición de datos.
    System.out.println("Introduce la cantidad de números primos que quieres mostrar: ");
    cantidad = s.nextInt();
    
    //Ciclo for y salida.
    //Debe mostrar con un bucle for una x cantidad de numeros a partir de 2 que es el primer número primo.
    for (int primo=2; primo<=cantidad;primo++) {
        //Si el numero primo es divisible por el mismo o por 1, se cumple que es primo y lo imprime.
      if ((primo%2!=0 || primo==2) && (primo%3!=0 || primo==3) && (primo%5!=0 || primo==5) && (primo%7!=0 || primo==7)){
        System.out.println(primo);
    }
  }
    
  }
}
